Formula/w3m.rb

```diff
@@ -1,26 +1,26 @@
 class W3m < Formula
   desc "Pager/text based browser"
   homepage "https://w3m.sourceforge.io/"
-  revision 6
+  revision 7
   head "https://github.com/tats/w3m.git"
 
   stable do
-    url "https://downloads.sourceforge.net/project/w3m/w3m/w3m-0.5.3/w3m-0.5.3.tar.gz"
+    url "https://deb.debian.org/debian/pool/main/w/w3m/w3m_0.5.3.orig.tar.gz"
     sha256 "e994d263f2fd2c22febfbe45103526e00145a7674a0fda79c822b97c2770a9e3"
 
     # Upstream is effectively Debian https://github.com/tats/w3m at this point.
     # The patches fix a pile of CVEs
     patch do
-      url "https://deb.debian.org/debian/pool/main/w/w3m/w3m_0.5.3-37.debian.tar.xz"
-      sha256 "625f5b0cb71bf29b67ad3bb9c316420922877473a6e94e6c7bcc337cb22ce1eb"
+      url "https://deb.debian.org/debian/pool/main/w/w3m/w3m_0.5.3-38.debian.tar.xz"
+      sha256 "227dd8d27946f21186d74ac6b7bcf148c37d97066c7ccded16495d9e22520792"
       apply "patches/010_upstream.patch",
             "patches/020_debian.patch"
     end
   end
 
   livecheck do
     url :stable
-    regex(%r{url=.*?/w3m[._-]v?(\d+(?:\.\d+)+)\.t}i)
+    regex(%r{url=.*?/w3m[._-]v?(\d+(?:\.\d+)+)\.orig.t}i)
```

Comment:
I'm not too familiar with the suggestions thing, but this should be:
```
  url "https://deb.debian.org/debian/pool/main/w/w3m/"
  regex(%r{href=.*?w3m[._-]v?(\d+(?:\.\d+)+)\.orig\.t}i)
```
This ensures `0.5.3` is still reported as the version found by livecheck.